apps/api-evidence/src/app/utils/types.py

```diff
@@ -69,11 +69,9 @@ def __get_validators__(cls) -> Generator[Callable[..., Any], None, None]:
     def validate(cls, v: Any) -> 'EvidenceFile':
         file: UploadFile = UploadFile.validate(v)
 
-        size_in_mb = 0.0
-        for chunk in file.file:
-            size_in_mb += len(chunk) / 1024 / 1024
-            if size_in_mb > config.MAX_FILE_SIZE:
-                raise FileTooLargeError
+        size_in_mb = sum([len(chunk) / 1024 / 1024 for chunk in file.file])
```

Comment:
I'm not sure if this is more readable than your original version, but I think we can keep it.